snippets/updating-data/create-record/in-place-body.js

```diff
@@ -7,7 +6,7 @@ const request = createRecord(record);
 request.body = JSON.stringify(
   serializeResources(
     store.cache,
-    recordIdentifierFor(record)
```

Comment:
here we do use recordIdentifierFor because we have the record and not the identifier.

The builders do this conversion for us when setting up the request options, thats why we don't need to redo this inside the handler example

Reply:
aah got it